protoc-gen-go/generator/name_test.go

```diff
@@ -87,36 +86,36 @@ func TestGoPackageOption(t *testing.T) {
 
 func TestUnescape(t *testing.T) {
 	tests := []struct {
-		in  string
-		out string
-		err string
+		in   string
+		out  string
+		fail bool
 	}{
 		// successful cases, including all kinds of escapes
-		{"", "", ""},
-		{"foo bar baz frob nitz", "foo bar baz frob nitz", ""},
-		{"\\000\\001\\002\\003\\004\\005\\006\\007", string([]byte{0, 1, 2, 3, 4, 5, 6, 7}), ""},
-		{"\\a\\b\\f\\n\\r\\t\\v\\\\\\?\\'\\\"", "\a\b\f\n\r\t\v\\?'\"", ""},
-		{"\\x10\\x20\\x30\\x40\\x50\\x60\\x70\\x80", "\x10\x20\x30\x40\x50\x60\x70\x80", ""},
+		{"", "", false},
+		{"foo bar baz frob nitz", "foo bar baz frob nitz", false},
+		{"\\000\\001\\002\\003\\004\\005\\006\\007", string([]byte{0, 1, 2, 3, 4, 5, 6, 7}), false},
```

Comment:
The double escaping is a little hard to read. Can you use raw-string literals?
```go
`\000\001\002\003\004\005\006\007`
```

Reply:
Done. When I did this, it made it harder to easily see the difference between inputs and expected outputs. In several cases, the only difference was the kind of quote mark, which seemed overly subtle when reading. So I changed expected outputs in a few cases to use a []byte instead of string, to make the expected decoding more obvious.